src/main/java/com/agilethought/internship/sso/controller/ControllerApplication.java

```diff
@@ -26,28 +36,33 @@
 	@PostMapping(value = "/users", produces = "application/json")
 	@ResponseStatus(value = HttpStatus.CREATED)
 	@ApiOperation(value = "Create New User in the application")
-	@ApiResponses(value = {
```

Comment:
For next PR and commits. please keep same format in previous code. Just execute re-format operation for your changes not all class.